### ISSUE TITLE
"_OBJC_CLASS_$_FIRAnalytics" from Firebase Analytics

### ISSUE THREAD

**Issue by efdalustaoglu** [chat-sdk/chat-sdk-ios#119]
When I try to call Firebase Analytics from the code;

`[FIRAnalytics setUserPropertyString:@“Identifier" forName:@“MyViewController"];`

I'm getting an error,

```
Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_FIRAnalytics", referenced from:
      objc-class-ref in MyVC.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Does anybody face that issue before, what am I missing? Can anyone help me please with this issue?

**Comment by simonsmiley64** [comment]
Hi @efdalustaoglu,

Can you confirm when you are having this error appear?

1. First, does the project compile and run when downloaded from Github, this is running on all our test accounts
2. Next, does it compile and run when you change the Firebase accounts to your own. It is also worth checking you have set it up correctly at this stage to ensure you are connecting to Firebase.
3. Have you made any modifications to the code? If yes what have you changed and where? Are you trying to add the Chat SDK in a different way place or form? This will help us troubleshoot the issues,

Let me know and we'll do our best to help,

Simon

**Comment by efdalustaoglu** [comment]
1. Yes it is running.
2. It also compiles (and I can register with my facebook/twitter/g+ accounts and am able to see my earlier messages as soon as I login). 
3. No I didn't change anything in the sdk.

I also tried to add this line in the sample project and I got the same error again. 

```
[BStorageManager sharedManager].a = [[BCoreDataManager alloc] init];
[FIRAnalytics setUserPropertyString:@"Identifier" forName:@"MyViewController"];
```

same for FirebaseAuth library.
`"_OBJC_CLASS_$_FIRPhoneAuthProvider"` 

I don't know if I'm calling it wrong or not but since I cannot add any Firebase libraries into my pod because ChatSDK handles it (I can see 'FirebaseAnalytics' & 'FirebaseAuth' frameworks under 'ChatSDKFirebaseAdapter') I assume they should work when I call them. However I'm getting

```
"_OBJC_CLASS_$_FIRAnalytics"
"_OBJC_CLASS_$_FIRPhoneAuthProvider"
```
 whenever I call the functions under those frameworks.

I'm also providing my 'ChatSDKFirebaseAdapter.podspec' if I need to make any configuration on it.
[ChatSDKFirebaseAdapter.txt](https://github.com/chat-sdk/chat-sdk-ios/files/1251130/ChatSDKFirebaseAdapter.txt)
